tests/perf/pubsub_bulk_subscribe_http/test.js

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import http from "k6/http";
+import { check } from "k6";
+import ws from "k6/ws";
+
+const targetUrl = __ENV.TARGET_URL
+const pubsubName = __ENV.PUBSUB_NAME
+const subscribeType = __ENV.SUBSCRIBE_TYPE
+const httpReqDurationThreshold = __ENV.HTTP_REQ_DURATION_THRESHOLD
+const defaultTopic = "perf-test"
+const defaultCount = 100
+const hundredBytesMessage = "a".repeat(100)
+
+export const options = {
+    discardResponseBodies: true,
+    thresholds: {
+        checks: ['rate==1'],
+        // 95% of requests should be below HTTP_REQ_DURATION_THRESHOLD milliseconds
+        http_req_duration: ['p(95)<' + httpReqDurationThreshold],
+    },
+    scenarios: {
+        bulkPSubscribe: {
+            executor: "ramping-vus",
```

Comment:
fyi: https://k6.io/docs/using-k6/scenarios/executors/ you can try different types of executors